daemon/info.go

```diff
@@ -174,7 +175,13 @@ func (daemon *Daemon) showPluginsInfo() types.PluginsInfo {
 
 	pluginsInfo.Volume = volumedrivers.GetDriverList()
 	pluginsInfo.Network = daemon.GetNetworkDriverList()
-	pluginsInfo.Authorization = daemon.configStore.AuthorizationPlugins
+
+	authPlugins := []string{}
```

Comment:
Thanks @tiborvass for your feedback. 

Now I updated the PR with the way you suggested.

While I think I wish to ask further the true reason why this is better. At the very beginning, I think both are OK. What I can think of is that maybe your way can reserved certain memory at the beginning, this will avoid some latter memory issue. If that, it is indeed important.

Could you please share more inner cause of that? So that I could avoid the same mistake in the future. Thanks a lot.